### PR TITLE
Fixed Archlinux docker build 

### PR DESCRIPTION
**Pull request by victorgveloso**
- Updated archlinux pkg format from old `.tar.xz` to the new standard `.tar.zst`
- fixed PKGBUILD src relative path usage

### CODE REVIEW

File: distros/ArchLinux/mkconfig.sh [hunk 1]
```diff
@@ -7,5 +7,5 @@ sed -i '/^post_install/r../../dist-utils/common/postinstall' "$DEST_FILE"
 sed -i '/^pre_remove/r../../dist-utils/common/preremove' "$DEST_FILE"
 sed -i '2,${/^#!/d}' "$DEST_FILE"
 
-export COPY_CMD='cp *.pkg.tar.xz /app/'
```
Comment (core1024) [inline distros/ArchLinux/mkconfig.sh:10]: The extension is set on purpose in order to be predictable an this way we can upload the package. If you change the extension it must be changed also here https://github.com/Stremio/stremio-shell/blob/master/.travis.yml#L64

Comment (victorgveloso) [reply distros/ArchLinux/mkconfig.sh:10]: Done. Thank you for the instructions!

**Review by core1024 — APPROVED** [review]

File: distros/ArchLinux/mkconfig.sh (hunk 1 above)
Comment (core1024) [reply distros/ArchLinux/mkconfig.sh:10]: Thank you for the PR. Obviously we have some issues with CI and the build fails. When we fix it we will merge your code.

**Comment by core1024** [comment]
I remembered that the extension should be changed on one more place - in [package.sh](https://github.com/victorgveloso/stremio-shell/blob/arch-build/distros/ArchLinux/package.sh#L8)

**Review by core1024 — APPROVED** [review]

**Comment by victorgveloso** [comment]
Sorry for the delay. I've been busy recently...

**Comment by core1024** [comment]
> Sorry for the delay. I've been busy recently...

No problem. We appreciate your time.

Unfortunately I'll need you to do one more thing. As your PR showed us that we have issues with the CI and as you can see the checks failed, I would kindly ask you to merge master, where the CI should be fixed, into your branch. This way the checks will pass and I will merge your pull request.

**Comment by victorgveloso** [comment]
I rebased my PR on the current master branch

**Comment by core1024** [comment]
> I rebased my PR on the current master branch

Thank you very much. Not only you fixed a bug but also by making a community PR we discovered and fixed bugs in our CI checks.

**Pull request merged**